Makefile.am

```diff
@@ -64,6 +64,7 @@ SINGA_SRCS := src/driver.cc \
               src/neuralnet/input_layer/record.cc \
               src/neuralnet/input_layer/deprecated.cc \
               src/neuralnet/input_layer/store.cc \
+              src/neuralnet/input_layer/dummy.cc \
```

Comment:
there is a DummyLayer in src/neuralnet/neuron_layer/dummy.cc.
can we merge these two layers @wangsheng1001 ?
I think their functionality should be similar.